ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -314,6 +327,34 @@ private class SqlLedger(
         UploadPackagesResult.Ok
       }(DEC)
   }
+
+  override def publishConfiguration(
+      maxRecordTime: Time.Timestamp,
+      submissionId: String,
+      config: Configuration): Future[SubmissionResult] =
+    enqueue { offsets =>
+      val recordTime = timeProvider.getCurrentTime
+      // FIXME(JM): Configurations with wrong generation or used submission id will result in 'Duplicate'.
+      // For the wrong generation case one would prefer storing a rejection!
+      ledgerDao
+        .storeConfigurationEntry(
+          offsets.offset,
+          offsets.nextOffset,
+          None,
+          recordTime,
+          submissionId,
+          participantId,
```

Comment:
why is the local participantId used here?

Reply:
Aehm, because you added it in `LedgerDao`?

Reply:
(this was a comment for myself to fix it :D)